.github/workflows/pullrequest.yml

```diff
@@ -0,0 +1,49 @@
+name: Pull Request
```

Comment:
log for pull request workflow

https://github.com/yshrsmz/android-client-sdk/actions/runs/3089683437

Reply:
It seems to be a bit faster than Bitrise, but the on-push workflow takes more than I imagined.